src/app/app.routes.js

```diff
@@ -36,6 +36,38 @@ angular.module('app-bootstrap').config([
             controllerAs: 'comp2Ctrl'
           }
         }
+      })
+      .state('centered.requiredLogin', {
+        url: '/required_login',
```

Comment:
add an abstract state with an authController and another abstract state with an unauthorizedController instead of this one